pythonforandroid/python.py

```diff
@@ -171,7 +171,7 @@ def get_recipe_env(self, arch=None, with_flags_in_cc=True):
         env['SYSROOT'] = sysroot
 
         if sh.which('lld') is not None:
-            # Note: The -L. is to fix a bug in python 3.7. 
+            # Note: The -L. is to fix a bug in python 3.7.
```

Comment:
Yes sorry :joy: 
And I still haven't enabled all the rules, but it may come soon or later :stuck_out_tongue: